### PR TITLE
Error.StorageConflict

### PR DESCRIPTION
**Pull request by jonajonlee**
Throw a standard Storage Conflict error in case of Couch Conflict Error

### CODE REVIEW

**Comment by jxbadam** [comment]
@jonajonlee You have to add tests for your new error.

File: lib/couchdb/index.js [hunk 1]
```diff
@@ -182,9 +185,11 @@ const deleteById = R.curry((couch, bucket, id) =>
 )
 
 
-module.exports = (server_config) =>
+module.exports = (server_config, wrapper = CouchWrapper) =>
+
+  wrapper(server_config)
 
-  CouchWrapper(server_config)
+  .then(x => {console.log('debugK000', x); return x})
```
Comment (jxbadam) [inline lib/couchdb/index.js]: console.log ? 🐟

**Review by jxbadam — CHANGES_REQUESTED** [review]
Hell is always on :fire:

File: lib/couchdb/index.js (hunk 1 above)
Comment (jonajonlee) [reply lib/couchdb/index.js]: it's gone!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

**Review by jxbadam — APPROVED** [review]
:shipit: :fire:

**Review by mattsmee — APPROVED** [review]
:shipit:

**Comment by jonajonlee** [comment]
:shipit:  by Michal

**Review by mszymulanski — APPROVED** [review]
:shipit: <- by me.

**Pull request merged**